opencti-platform/opencti-front/src/components/ItemParticipants.tsx

```diff
@@ -1,28 +1,65 @@
 import React, { FunctionComponent } from 'react';
-import Button from '@mui/material/Button';
+import { useTheme } from '@mui/styles';
+import { CancelOutlined, PersonOutline } from '@mui/icons-material';
+import Chip from '@mui/material/Chip';
+import { stixDomainObjectMutation } from '@components/common/stix_domain_objects/StixDomainObjectHeader';
+import Tooltip from '@mui/material/Tooltip';
 import FieldOrEmpty from './FieldOrEmpty';
+import type { Theme } from './Theme';
+import useGranted, { KNOWLEDGE_KNUPDATE } from '../utils/hooks/useGranted';
+import { truncate } from '../utils/String';
+import { commitMutation, defaultCommitMutation } from '../relay/environment';
 
 interface ItemParticipantsProps {
   participants: {
     readonly entity_type: string
     readonly id: string
     readonly name: string
   }[];
+  stixDomainObjectId: string;
 }
 
-const ItemParticipants: FunctionComponent<ItemParticipantsProps> = ({ participants }) => {
+const ItemParticipants: FunctionComponent<ItemParticipantsProps> = ({ participants, stixDomainObjectId }) => {
+  const theme = useTheme<Theme>();
+  const canUpdateKnowledge = useGranted([KNOWLEDGE_KNUPDATE]);
+  const handleRemoveParticipant = (removedId: string) => {
+    const values = participants.filter((participant) => participant.id !== removedId);
+    const valuesIds = values.map((value) => value.id);
+    commitMutation({
+      mutation: stixDomainObjectMutation,
+      variables: {
+        id: stixDomainObjectId,
+        input: {
+          key: 'objectParticipant',
+          value: valuesIds,
+        },
+      },
+      ...defaultCommitMutation,
+    });
+  };
   return (
     <FieldOrEmpty source={participants}>
       {participants.map((participant) => (
-        <Button
-          key={participant.id}
-          variant="outlined"
-          color="primary"
-          size="small"
-          style={{ margin: '0 7px 7px 0', cursor: 'default' }}
-        >
-          {participant.name}
-        </Button>
+        <Tooltip key={participant.id} title={participant.name}>
+          <Chip
+            key={participant.id}
+            variant="outlined"
+            icon={<PersonOutline color={'primary'} />}
+            label={truncate(participant.name, 25).toUpperCase()}
+            style={{
```

Comment:
better to include the styles in a 'useStyles' constant out of the component definition

Reply:
Ok, but above all useStyles we have added a comment wich say : Don't use for new code. I thought good practice was like this, inline style.

Reply:
mm indeed, sorry, I don't know best practices for css